packages/nextjs/components/BatchTimeline.tsx

```diff
@@ -0,0 +1,229 @@
+"use client";
```

Comment:
We don't need this restriction since this component doesn't use any client-only mechanism. Please remove this line so we can improve the app's performance by pre-rendering the component during the compile process.